src/main/kotlin/services/AssignmentService.kt

```diff
@@ -51,7 +50,8 @@ class AssignmentService(
     }
 
     fun findAllActiveByPerson(from: LocalDate, to: LocalDate, personCode: UUID): List<Assignment> {
-        val query = "SELECT a FROM Assignment a WHERE a.from <= :to AND (a.to is null OR a.to >= :from) AND a.person.uuid = :personCode"
+        val query =
+            "SELECT a FROM Assignment a WHERE a.from <= :to AND (a.to is null OR a.to >= :from) AND a.person.uuid = :personCode"
         return entityManager
             .createQuery(query, Assignment::class.java)
             .setParameter("from", from)
```

Comment:
This looks like an accidental touch by reformat. Bit since you touched it, can you boyscout this and move the implementation to the repository, and mimick the others methods in this class?

Reply:
Good point, I will move all custom queries from Assignment service to the repository layer